test/controllers/test_case_controller.py

```diff
@@ -21,6 +23,12 @@ def test_get_all_business_ids_for_collection_exercise(self, mock_engine):
         test_output = case_controller.get_all_business_ids_for_collection_exercise(EXERCISE_ID)
         self.assertEqual(expected_output, test_output)
 
+    def test_get_all_business_ids_for_collection_exercise_failure(self):
+        expected_output = None
+        test_output = case_controller.get_all_business_ids_for_collection_exercise("bad_exercise_id")
```

Comment:
While this does raise a SQLAlchemyError you could also do it by patching the method and using `.side_effect` to raise the error

Reply:
I ended up encasing the tests inside a `with assertRaises(SQLAlchemyError):` block.